### DISCUSSION TITLE
关于自动选人的问题

### DISCUSSION THREAD

**Discussion by yblkksk** [HSLix/LixAssistantLimbusCompany#93] [General]
看到选了二灯人格，停止脚本之后调整了人格选择，但是进去之后还是会重新选一遍选到二灯人格去。

**Comment by yblkksk** [comment]
试了一下编辑sinners_order.txt的解决方案，打了一会后面还是和以前一样开始乱选了

**Comment by HSLix** [comment]
请提供sinners_order.txt和相关情况的录像视频（可以发b站什么的）以判断问题。另外你说会选到二灯人格，LALC 暂不支持区分二/三灯人格，只能调整选人先后顺序

**Comment by yblkksk** [comment]
[
[sinner_order.txt](https://github.com/user-attachments/files/16391000/sinner_order.txt)
](url)

**Comment by yblkksk** [comment]
stop后再启动脚本后倒是正常了

**Comment by yblkksk** [comment]
问题发生在自动刷完一把之后的第二把，稍后会录个屏

**Comment by yblkksk** [comment]
要不开发者建个群吧，tg或qq都行，这里发录屏得压缩一遍，有10MB的大小限制

**Comment by yblkksk** [comment]
①在一次启动中第一把结束之后进入镜牢战斗默认会选择几个人格，但是没选满，此时脚本会补位选满，但其实要全部取消再点才是正常的，这个问题有偶发性，有时候首次战斗默认是正确的顺序②当配置文件里的前六个罪人中有罪人阵亡时，此时队伍人数只有五人，下次战斗开始前的队伍编辑会把罪人全部取消然后按照配置文件的顺序从第七名罪人开始进行选择。

**Reply by HSLix** [reply]
> ①在一次启动中第一把结束之后进入镜牢战斗默认会选择几个人格，但是没选满，此时脚本会补位选满，但其实要全部取消再点才是正常的，这个问题有偶发性，有时候首次战斗默认是正确的顺序②当配置文件里的前六个罪人中有罪人阵亡时，此时队伍人数只有五人，下次战斗开始前的队伍编辑会把罪人全部取消然后按照配置文件的顺序从第七名罪人开始进行选择。

在当前版本的 LALC 中，这两个特性是正常的，没有考虑到阵亡的情况，毕竟在陆爻齐的使用下来，罪人阵亡是极小概率事件，即使发生，也几乎不影响打通普通难度的镜牢四。

不过从设计角度而言，确实算 bug，还是会修复的

**Reply by yblkksk** [reply]
好的，辛苦开发者

**Reply by yblkksk** [reply]
因为刚入坑不到一周，box欠佳，有时候自动p会翻车（
